code/game/machinery/recharger.dm

```diff
@@ -88,9 +109,10 @@
 			var/obj/item/gun/energy/E = charging
 			if(E.power_supply.charge < E.power_supply.maxcharge)
 				E.power_supply.give(E.power_supply.chargerate)
-				use_power(250)
+				use_power(250 * recharge_coeff)
```

Comment:
...And rating of capacitor increases the better they are...

Wouldn't this line results in the recharge using up more energy?

Reply:
very smooth by me, I multiplied the coefficient on the wrong thing.